tools/harness-thci/OpenThread.py

```diff
@@ -3123,23 +3124,26 @@ def __detectZephyr(self):
 
     def __discoverDeviceCapability(self):
         """Discover device capability according to version"""
-        if self.IsBorderRouter:
+        thver = self.__executeCommand('thread version')[0]
+        if thver in ['1.3', '4'] and not self.IsBorderRouter:
```

Comment:
```suggestion
        if thver in ['1.3', '4'] and self.IsBorderRouter:
```

Reply:
I'm not seeing a reason for this change.  The logic is correct as written as far as I can tell.  I believe I have covered both cases where this was a 1.3 device and border router as well as where it is not (one right after the other).  Can you comment a little on what you are looking for here?  Thanks.